src/components/modals/EntityExtractor.tsx

```diff
@@ -58,7 +60,8 @@ class EntityExtractor extends React.Component<Props, ComponentState> {
             savedRoundIndex: 0,
             textVariationValue: '',
             newTextVariations: [],
-            entityTypeFilter: CLM.EntityType.LUIS
+            entityTypeFilter: CLM.EntityType.LUIS,
+            openPickerText: null
```

Comment:
Don't think the name of this is clear.  Is there a better way to describe it's use?

Initially I would think it sets the text of the open picker, but it seems to be actually just a copy of the latest extraction text for comparison.

Reply:
"activePickerText"?

Reply:
I don't think that helps the situation. "open" and "active" are essentially the same thing.  The picker has it's own text and the extraction has text.

My understanding is that you only want to leave the picker open for the extractions that have the matching text. All those that don't match should be closed.

Would prefer
`openExtractionText`